WebContent/WEB-INF/jsp/collectionImport.jsp

```diff
@@ -73,7 +64,7 @@
             }
         }
         
-        var message = "Are you sure you want to Create collection '" + collTitle + "'? \n";
+        var message = "Are you sure you want to Create new collection? \n";
```

Comment:
Can this read "Are you sure you want to Create a new collection?" Basically add an "a" between Create and new.